apps/voting/app/src/math-utils.js

```diff
@@ -38,10 +39,88 @@ export function formatNumber(num, decimals = 2, { truncate = true } = {}) {
     : formattedNumber
 }
 
-export function percentageList(values, digits = 0) {
+export function percentageList(values) {
   return scaleBNValuesSet(values).map(value => value.toNumber())
 }
 
+// Format a percentage from BN values.
```

Comment:
Can you add to those functions standard JSDoc annotations? It will increase the readability.

This is how I normally do them: 
https://github.com/aragon/govern/blob/feature/govern-tx/packages/govern-tx/src/wallet/Wallet.ts#L36-L48

Reply:
I think the FuncDoc only got added in one place. :-)

Reply:
You're right, my bad :)